extensions/helix/package.build.ts

```diff
@@ -274,6 +274,225 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
+            command: "workbench.action.debug.start",
+          },
+          "w": {
+            text: "Window",
+            command: "dance.openMenu",
+            args: [{ menu: "window" }],
+          },
+          "y": {
+            text: "Yank selections to clipboard",
+            command: "dance.selections.saveText",
+            args: [{
+              register: "",
+            }],
+          },
+          "Y": {
+            text: "Yank main selection to clipboard",
+            command: "dance.selections.saveText",
+            args: [{
+              register: "",
+            }],
+          },
+          "p": {
+            text: "Paste clipboard after selections",
+            command: "dance.edit.insert",
+            args: [{
+              handleNewLine: true,
+              where: "end",
+            }],
+          },
+          // The following comment was left by https://github.com/Silverquark
+          // Not sure if it’s still needed.
+          //
+          // There is a zero width space (U+200B) behind the P.
+          // This is a dirty hack. Otherwise vscode will think its the same as lowecase p
+          // Any other symbol would also work, but this one is invisible
+          "P​": {
+            text: "Paste clipboard before selections",
+            command: "dance.edit.insert",
+            args: [{
+              handleNewLine: true,
+              where: "start",
+            }],
+          },
+          "R": {
+            text: "Replace selections by clipborad content",
+            command: "editor.action.clipboardPasteAction",
+            args: [],
+          },
+          "/": {
+            text: "Global Search in workspace folder",
+            command: "workbench.action.findInFiles",
+          },
+          "k": {
+            text: "Show docs for item under cursor (hover)",
+            command: "editor.action.showHover",
+          },
+          "r": {
+            text: "Rename symbol",
+            command: "editor.action.rename",
+          },
+          "h": {
+            text: "Select symbol reference",
+            command: "editor.action.referenceSearch.trigger",
+          },
+        },
+
+        "window": {
+          title: "Window",
+          items: {
+            "w": {
+              text: "Goto next window",
+              command: "workbench.action.nextEditor",
+            },
+            "s": {
+              text: "Horizontal bottom split",
+              command: "workbench.action.splitEditorDown",
+            },
+            "v": {
+              text: "Vertical right split",
+              command: "workbench.action.splitEditor",
+            },
+            "t": {
+              text: "Transpose splits",
+              command: "workbench.action.toggleEditorGroupLayout",
+            },
+            // "f": {
+            //   text: "Open files in selection (hsplit)",
+            //   command: "dance.selections.open", function needs to be modified
```

Comment:
nit: to make it easier to work on this by uncommenting, this should be valid code

```suggestion
            //   command: "dance.selections.open", // function needs to be modified
```